lisa/operating_system.py

```diff
@@ -330,7 +330,16 @@ def _get_information(self) -> OsInformation:
             if os_release_info.group("name") == "NAME":
                 vendor = os_release_info.group("value")
             elif os_release_info.group("name") == "VERSION_ID":
-                release = os_release_info.group("value")
+                # get release from /etc/debian_version for debian distro
+                # e.g. credativ debian 9-backports 9.20190313.0
+                # cat /etc/debian_version - 9.8
+                # VERSION_ID="9"
+                if "Debian" in vendor:
```

Comment:
The logic should be in Debian, instead of posix.

Reply:
Fixed